app/models/source_file.rb

```diff
@@ -61,4 +63,26 @@ def self.recently_redacted(start_time: Time.zone.yesterday.beginning_of_day, end
       )
     )
   end
+
+  def self.pdf_attachment
+    includes(active_storage_attachment: :blob).where(
+      active_storage_attachment: {
+        active_storage_blobs: {
+          content_type: PDF_CONTENT_TYPE
+        }
+      }
+    )
+  end
+
+  def self.not_redacted
+    includes(:redacted_source_file_attachment).where(
+      redacted_source_file_attachment: {
+        id: nil
+      }
+    )
+  end
```

Comment:
Can this method and the one above be private methods?

Reply:
@littleforest I tried this suggestion, but it's breaking the implementation. My guess is that combining scopes (`completed` from the enum) and private class methods does not work together well 🤔